localstack-core/localstack/aws/api/pipes/__init__.py

```diff
@@ -231,8 +231,6 @@ class TimeFieldType(StrEnum):
 
 
 class ConflictException(ServiceException):
-    """An action you attempted resulted in an exception."""
```

Comment:
nit: It seems we are using comments in ext but not in LocalStack community because these APIs here were simply moved from ext in https://github.com/localstack/localstack/pull/11231
Is that intended @alexrashed ?

Reply:
Yes, that's intended, because we want to avoid the comments bloating up the package size.
This isn't a probpem for our proprietary packages because we minify them, removing all comments when creating the packages.